README.md

```diff
@@ -61,7 +61,7 @@ $ NAMESPACE=`oc project -q`
 $ sed -i'' "s/namespace:.*/namespace: $NAMESPACE/g" ./deploy/rbac.yaml
 $ oc create -f deploy/rbac.yaml
 $ oc create role use-scc-hostmount-anyuid --verb=use --resource=scc --resource-name=hostmount-anyuid -n $NAMESPACE
```

Comment:
does this command work for you?

Reply:
Yes, the `oc create role...` command completed, but the Provisioner pod was stuck unable to provision storage until I added the SCC to the ServiceAccount.

```
[jcall@jcall-laptop ~]$ export NAMESPACE=nfs-storage

[jcall@jcall-laptop ~]$ oc get role -n $NAMESPACE
NAME                                    CREATED AT
leader-locking-nfs-client-provisioner   2021-03-04T19:03:12Z
use-scc-hostmount-anyuid                2021-03-04T19:03:20Z

[jcall@jcall-laptop ~]$ oc get role/use-scc-hostmount-anyuid -n $NAMESPACE -o yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  creationTimestamp: "2021-03-04T19:03:20Z"
  managedFields:
  - apiVersion: rbac.authorization.k8s.io/v1
    fieldsType: FieldsV1
    fieldsV1:
      f:rules: {}
    manager: kubectl-create
    operation: Update
    time: "2021-03-04T19:03:20Z"
  name: use-scc-hostmount-anyuid
  namespace: nfs-storage
  resourceVersion: "103774693"
  selfLink: /apis/rbac.authorization.k8s.io/v1/namespaces/nfs-storage/roles/use-scc-hostmount-anyuid
  uid: 0ed79b63-bcc6-45ac-8986-07fab4639758
rules:
- apiGroups:
  - security.openshift.io
  resourceNames:
  - hostmount-anyuid
  resources:
  - securitycontextconstraints
  verbs:
  - use
```

Reply:
for me in 3.11 it throws error that i can't use `use` on this resource exactly like #60. 
do you know how to make the readme compatible with both v3 and v4 versions?